lib/vmdb/loggers.rb

```diff
@@ -76,6 +80,37 @@ def self.create_logger(log_file_name, logger_class = VMDBLogger)
       nil
     end
 
+    private_class_method def self.progname_from_file(log_file_name)
+      File.basename(log_file_name, ".*")
+    end
+
+    private_class_method def self.create_wrapper_logger(log_file_name, logger_class, wrapped_logger)
+      log_file_name = Pathname.new(log_file_name) unless log_file_name.kind_of?(Pathname)
+      progname      = progname_from_file(log_file_name)
+
+      logger_class.new(nil, progname: progname).tap do |logger|
```

Comment:
the hash rocket is dead.
Long live the hash rocket

Reply:
Nope, this is a keyword param 😉  - I've always said that I liked this syntax for keyword params but not regular Hashes.